lib/tdf3/src/tdf.ts

```diff
@@ -145,6 +145,7 @@ export type EncryptConfiguration = {
   keyForEncryption: KeyInfo;
   keyForManifest: KeyInfo;
   assertionConfigs?: AssertionConfig[];
+  targetSpecVersion?: string;
```

Comment:
Let's see if I'm understanding this correctly. 

When hex encoding was removed in this pr it was relying on the spec version to be `3.0.0`
https://github.com/opentdf/web-sdk/pull/397

Here how it was hardcoded
https://github.com/opentdf/web-sdk/blob/6ca50e6f3b2d945c98993190784a78e960438d8f/lib/tdf3/src/tdf.ts#L261

And updated  to use schema version and not tdf version 
https://github.com/opentdf/web-sdk/pull/414

Reply:
So I am wondering, if the flag  `isLegacyTDF` correct.
https://github.com/opentdf/web-sdk/blob/a240d66de226f0420486d25c0c904ce2b22bac63/lib/tdf3/src/tdf.ts#L964

Is it reliable  that only tdf with manifest schema as `3.0.0` have been encoded with hex signature hash?

(Depending on how the release was cut, we might have encrypted tdf manifest with and without the hex on the version `3.0.0`)

Reply:
I updated the flag to use  `semver.lt(specVersion, '4.3.0`) as cutout